src/operator-docs/creation/timer.ts

```diff
@@ -1,6 +1,65 @@
 import { OperatorDoc } from '../operator.model';
 
 export const timer: OperatorDoc = {
-  'name': 'timer',
-  'operatorType': 'creation'
+  name: 'timer',
+  operatorType: 'creation',
+  signature:
+    'public timer(initialDelay: number | Date, period : number = 0, scheduler: IScheduler = async): Observable<number>',
```

Comment:
could you remove the generic as described in the contribution.md

Reply:
Removed in recent commit [46b699f](https://github.com/ReactiveX/rxjs-docs/pull/299/commits/46b699fa2faee3c5cc58bf21c52aa5eebd449fd7)